gtk3/sample/misc/label.rb

```diff
@@ -1,23 +1,21 @@
 #!/usr/bin/env ruby
 =begin
-  label.rb - Ruby/GTK2 sample script.
+  label.rb - Ruby/GTK3 sample script.
 
-  Copyright (c) 2002-2006 Ruby-GNOME2 Project Team
+  Copyright (c) 2002-2015 Ruby-GNOME2 Project Team
   This program is licenced under the same licence as Ruby-GNOME2.
-
-  $Id: label.rb,v 1.8 2006/06/17 13:18:12 mutoh Exp $
 =end
 
-require 'gtk3'
+require "gtk3"
 
 win = Gtk::Window.new("Gtk::Label sample")
 win.signal_connect("destroy"){Gtk.main_quit}
 label1 = Gtk::Label.new("_Not Use Mnemonic")
 label1.selectable = true
 label2 = Gtk::Label.new("_Use Mnemonic", true)
-label3 = Gtk::Label.new.set_markup(%Q[<markup><span size="xx-large" weight="bold" foreground="blue"><u>Ruby-GNOME2</u></span> is <span foreground="red" size="x-large"><i>cool</i></span>!</markup>], true)
+label3 = Gtk::Label.new.set_markup(%Q[<markup><span size="xx-large" weight="bold" foreground="blue"><u>Ruby-GNOME2</u></span> is <span foreground="red" size="x-large"><i>cool</i></span>!</markup>])
```

Comment:
Please use `Gtk::Label#set_markup(text, :use_underline => true)`. I've added it at master.